src/lib/components/Environment.svelte

```diff
@@ -0,0 +1,101 @@
+<script lang="ts">
+  import { Button } from "$lib/components/ui/button";
+  import * as Dialog from "$lib/components/ui/dialog";
+  import { Trash2 } from "lucide-svelte";
```

Comment:
I don't know why but this name made me laugh 😂

Reply:
I found it funny too!